Dockerfile

```diff
@@ -49,18 +26,15 @@ ENV     ACAS_BASE /home/runner/acas
 ENV     ACAS_CUSTOM /home/runner/acas_custom
 ENV     ACAS_SHARED /home/runner/acas_shared
 ENV     APACHE Redhat
-RUN     npm install -g gulp@4.0.0 forever nodemon mocha coffeescript
-COPY    package.json $ACAS_BASE/package.json
-RUN     chown -R runner:runner $ACAS_BASE
+RUN     npm install -g gulp@4.0.2 forever@3.0.4 coffeescript@2.5.1
+COPY    --chown=runner:runner package.json $ACAS_BASE/package.json
```

Comment:
Docker now allows you to ChOWN a file as you copy it (instead of making unnecessary layers in the Docker image by running "RUN chown" command")